### PR TITLE
Doc fix minor issues in intro

### PR DESCRIPTION
**Pull request by richardkapolnai-da**

### CODE REVIEW

File: docs/source/daml/intro/7_Composing.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ You can start a new project with a skeleton structure using ``daml new project_n
 
 You can generally set ``name`` and ``version`` freely to describe your project. ``dependencies`` does what the name suggests: It includes dependencies. You should always include ``daml-prim`` and ``daml-stdlib``. The former contains internals of compiler and DAML Runtime, the latter gives access to the DAML Standard Library. ``daml-script`` contains the types and standard library for DAML Script.
 
-You compile a DAML project by running ``daml build`` from the project root directory. This creates a ``dar`` file in ``.daml/dist/dist/project_name-project_version.dar``. A ``dar`` file is DAML's equivalent of a ``JAR`` file in Java: it's the artifact that gets deployed to a ledger to load the package and its dependencies. ``dar`` files are fully self-contained in that they contain all dependencies of the main package. More on all of this in :doc:`8_Dependencies`.
```
Comment (cocreature) [inline docs/source/daml/intro/7_Composing.rst:35]: Why did you remove the statement on DARs being self-contained?

**Review by cocreature — APPROVED** [review]
Thanks! One minor question on the removal of DARs being self-contained

File: docs/source/daml/intro/7_Composing.rst (hunk 1 above)
Comment (richardkapolnai-da) [reply docs/source/daml/intro/7_Composing.rst:35]: Because it was a misunderstanding on my part! Good catch! Reverting this change...

File: docs/source/daml/intro/7_Composing.rst [hunk 2]
```diff
@@ -32,7 +32,7 @@ You can start a new project with a skeleton structure using ``daml new project_n
...
+You compile a DAML project by running ``daml build`` from the project root directory. This creates a ``dar`` file in ``.daml/dist/dist/project_name-project_version.dar``. A ``dar`` file is DAML's equivalent of a ``JAR`` file in Java: it's the artifact that gets deployed to a ledger to load the package and its dependencies. More on all of this in :doc:`8_Dependencies`.
```
Comment (richardkapolnai-da) [inline docs/source/daml/intro/7_Composing.rst:35]:
```suggestion
You compile a DAML project by running ``daml build`` from the project root directory. This creates a ``dar`` file in ``.daml/dist/dist/project_name-project_version.dar``. A ``dar`` file is DAML's equivalent of a ``JAR`` file in Java: it's the artifact that gets deployed to a ledger to load the package and its dependencies. ``dar`` files are fully self-contained in that they contain all dependencies of the main package. More on all of this in :doc:`8_Dependencies`.
```
